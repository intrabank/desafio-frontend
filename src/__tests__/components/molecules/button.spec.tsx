import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Button from '../../../components/molecules/button';

describe('Button', () => {
	it('should render button component', () => {
		const tree = renderer.create(<Button>Button text</Button>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
