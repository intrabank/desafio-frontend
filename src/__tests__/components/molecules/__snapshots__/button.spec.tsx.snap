// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render button component 1`] = `
<button
  className="c-dInhNt"
  type="submit"
>
  Button text
</button>
`;
